lib/tmx-parser/listener.rb

```diff
@@ -0,0 +1,80 @@
+# encoding: UTF-8
+
+module TmxParser
+
+  class Listener
+    include TagNames
+
+    attr_reader :units, :proc
```

Comment:
Huh... a quick Google search didn't reveal that, but it totally works in irb. Thanks for the #protip!